solarkraft/test/unit/verify.test.ts

```diff
@@ -0,0 +1,170 @@
+// an example unit test to copy from
+
+import { assert } from 'chai'
+import { describe, it } from 'mocha'
+
+import { instrumentMonitor } from '../../src/verify.js'
+
+describe('verify command', () => {
+    it('instruments TLA+ monitors', () => {
+        const monitor = { modules: [{ declarations: [] }] }
+        const state = [
+            { name: 'is_initialized', type: 'TlaBool', value: false },
+        ]
+        const tx = {
+            functionName: 'Claim',
+            functionArgs: [{ type: 'TlaStr', value: 'alice' }],
+            env: { timestamp: 100 },
+            error: 'contract is not initialized',
+        }
+
+        const instrumented = instrumentMonitor(monitor, state, tx)
+        const expected = {
+            modules: [
+                {
+                    declarations: [
+                        {
+                            kind: 'TlaOperDecl',
+                            name: 'Init',
+                            type: 'Untyped',
+                            formalParams: [],
```

Comment:
These JSONs are usually exploding pretty fast in tests. Would it make sense to save the expected output into a file?

Reply:
Hmmm, I don't want to read from disk in a unit test, but I've moved it to a separate module: [76d8ce1](https://github.com/freespek/solarkraft/pull/39/commits/76d8ce12f3fb15c2d1e0e5dec6be29b445f4d38b)